BaseClasses.py

```diff
@@ -109,7 +109,7 @@ def set_player_attr(attr, val):
             set_player_attr('can_access_trock_front', None)
             set_player_attr('can_access_trock_big_chest', None)
             set_player_attr('can_access_trock_middle', None)
-            set_player_attr('fix_fake_world', True)
+            set_player_attr('fix_fake_world', logic[player] not in ['owglitches', 'nologic'] or shuffle[player] in ['crossed', 'insanity', 'madness_legacy'])
```

Comment:
Crossed and Insanity ER allows Fake World now? I'm not sure about that.

Reply:
It's the other way.  Crossed and insanity fix fake worlds even with glitch logic.